src/functionHandlers/notarisePdt/handler.ts

```diff
@@ -75,6 +85,19 @@ export const main: Handler = async (
 
   try {
     const result = await notarisePdt(reference, certificate);
+
+    /* Notify recipient via SPM */
```

Comment:
Could this be extracted into a separate try/catch block from the line above, which gets the notarised certificate url?

This is so that any failure to send the SPM notification would not also cause the API call to fail; at worst, it should only log an error internally but should not affect the consumer of the API.

Reply:
Oh yes, that's a good point. Let me separate it.